src/Plugin_Command.php

```diff
@@ -941,29 +941,70 @@ public function install( $args, $assoc_args ) {
 	 *   - yaml
 	 * ---
 	 *
+	 * ## AVAILABLE FIELDS
+	 *
+	 * These fields will be displayed by default for the plugin:
+	 *
+	 * * name
+	 * * title
+	 * * author
+	 * * version
+	 * * description
+	 * * status
+	 *
+	 * These fields are optionally available:
+	 *
+	 * * requires_wp
+	 * * requires_php
+	 * * requires_plugins
+	 *
 	 * ## EXAMPLES
 	 *
+	 *     # Get plugin details.
 	 *     $ wp plugin get bbpress --format=json
-	 *     {"name":"bbpress","title":"bbPress","author":"The bbPress Contributors","version":"2.6-alpha","description":"bbPress is forum software with a twist from the creators of WordPress.","status":"active"}
+	 *     {"name":"bbpress","title":"bbPress","author":"The bbPress Contributors","version":"2.6.9","description":"bbPress is forum software with a twist from the creators of WordPress.","status":"active"}
 	 */
 	public function get( $args, $assoc_args ) {
+		$default_fields = array(
+			'name',
+			'title',
+			'author',
+			'version',
+			'description',
+			'status',
+		);
+
 		$plugin = $this->fetcher->get_check( $args[0] );
 		$file   = $plugin->file;
 
 		$plugin_data = get_plugin_data( WP_PLUGIN_DIR . '/' . $file, false, false );
 
 		$plugin_obj = (object) [
-			'name'        => Utils\get_plugin_name( $file ),
-			'title'       => $plugin_data['Name'],
-			'author'      => $plugin_data['Author'],
-			'version'     => $plugin_data['Version'],
-			'description' => wordwrap( $plugin_data['Description'] ),
-			'status'      => $this->get_status( $file ),
+			'name'             => Utils\get_plugin_name( $file ),
+			'title'            => $plugin_data['Name'],
+			'author'           => $plugin_data['Author'],
+			'version'          => $plugin_data['Version'],
+			'description'      => wordwrap( $plugin_data['Description'] ),
+			'status'           => $this->get_status( $file ),
+			'requires_wp'      => '',
+			'requires_php'     => '',
+			'requires_plugins' => '',
 		];
 
+		$require_fields = [
+			'requires_wp'      => 'RequiresWP',
+			'requires_php'     => 'RequiresPHP',
+			'requires_plugins' => 'RequiresPlugins',
+		];
+
+		foreach ( $require_fields as $field_key => $data_key ) {
+			if ( isset( $plugin_data[ $data_key ] ) && ! empty( $plugin_data[ $data_key ] ) ) {
+				$plugin_obj->{$field_key} = $plugin_data[ $data_key ];
+			}
+		}
```

Comment:
Can we simplify this?

```suggestion
			'name'             => Utils\get_plugin_name( $file ),
			'title'            => $plugin_data['Name'],
			'author'           => $plugin_data['Author'],
			'version'          => $plugin_data['Version'],
			'description'      => wordwrap( $plugin_data['Description'] ),
			'status'           => $this->get_status( $file ),
			'requires_wp'      =>  ! empty( $plugin_data['requires_wp'] ) ? $plugin_data['requires_wp'] : '',
			'requires_php'     => ! empty( $plugin_data['requires_php'] ) ? $plugin_data['requires_php'] : '',
			'requires_plugins' => ! empty( $plugin_data['requires_plugins'] ) ? $plugin_data['requires_plugins'] : '',
		];
```